denizen_scripts/hub1/repo-link/systems/teleport.dsc

```diff
@@ -0,0 +1,1155 @@
+            - else:
+                - yaml id:teleporthere_cooldowns set <player.uuid>:<[uuid_here_list].map_with[<list.pad_right[<[uuid_here_list].size>].with[<util.time_now>]>]>
+            - narrate "<player.display_name><&a> has declined your teleport request." targets:<[uuid_list].include[<[uuid_here_list]>].deduplicate.parse_tag[<player[<[parse_value]>]>]>
+            - narrate "<&a>You have declined the teleport requests from: <[uuid_list].include[<[uuid_here_list]>].deduplicate.parse_tag[<player[<[parse_value]>].display_name>].formatted><&a>."
+        - else:
+            - define player <server.match_player[<context.args.first>]||null>
+            - if <[player]> == null:
+                - define Reason "The specified player is invalid."
+                - inject Command_Error
+            - if <yaml[teleport_requests].contains[<[player].uuid>]> && <yaml[teleport_requests].read[<[player].uuid>].contains[<player.uuid>]>:
+                - yaml set id:teleport_requests <[player].uuid>:<-:<player.uuid>
+                - if <yaml[teleport_cooldowns].contains[<[player].uuid>]>:
+                    - yaml id:teleport_cooldowns set <[player].uuid>:<yaml[teleport_cooldowns].read[<[player].uuid>].with[<player.uuid>].as[<util.time_now>]>
+                - else:
+                    - yaml id:teleport_cooldowns set <[player].uuid>:<map.with[<player.uuid>].as[<util.time_now>]>
+                - narrate "<player.display_name><&a> has declined your teleport request." targets:<[player]>
+                - narrate "<&a>You have declined <[player].display_name><&a>'s teleport request."
+            - else if <yaml[teleporthere_requests].contains[<player.uuid>]> && <yaml[teleporthere_requests].read[<player.uuid>].contains[<[player].uuid>]>:
+                - yaml set id:teleporthere_requests <player.uuid>:<-:<[player].uuid>
+                - if <yaml[teleporthere_cooldowns].contains[<player.uuid>]>:
+                    - yaml id:teleporthere_cooldowns set <player.uuid>:<yaml[teleporthere_cooldowns].read[<player.uuid>].with[<[player].uuid>].as[<util.time_now>]>
+                - else:
+                    - yaml id:teleporthere_cooldowns set <player.uuid>:<map.with[<[player].uuid>].as[<util.time_now>]>
+                - narrate "<player.display_name><&a> has declined your teleport request." targets:<[player]>
+                - narrate "<&a>You have declined <[player].display_name><&a>'s teleport request."
+            - else:
+                - define Reason "You do not have a teleport request from the specified player."
+                - inject Command_Error
+    - else:
+        # % ██ [  Syntax Check (Arg Count)  ] ██
+        - if <context.args.is_empty> || <context.args.size> > 2:
+            - inject Command_Syntax
+        - bungeerun hub1 networkteleport_deny def:<context.args.first>|<player>|<bungee.server>|<player.display_name>
```

Comment:
This type of list won't preserve each entry here if someone types something unexpected
```suggestion
        - bungeerun hub1 networkteleport_deny def:<list_single[<context.args.first>].include[<player>|<bungee.server>|<player.display_name>]>
```